dojo/tools/aqua/parser.py

```diff
@@ -55,43 +55,6 @@ def get_dedupe_fields(self) -> list[str]:
             "component_version",
         ]
 
-    # Jino This get_fields was written for the Aque Parser v2 (based off of "get_iten_v2")
-    # What do we do with the seperate versions of this parser?
-    # def get_fields(self) -> list[str]:
-    #     """
-    #     Return the list of fields used in the Aqua Parser V2
-    #
-    #     Fields:
-    #     - title: Created by combining the finding's cve and file_path
-    #     - description: Text describing finding
-    #     - url: Url associated with the finding
-    #     - severity: Severity rating converted from Aqua's integer format into DefectDojo's format.
-    #       #Jino: On line 106 it calls severity_of instead of aqua_severity_of. get_item v1 uses aqua_severity_of#
-    #     - impact: Impact rating of finding. Same as the finding severity.
-    #     - mitigation: If solution is true, mitigation equals true. If fix_version is true, mitigation equals 'Upgrade to True'.If neither are true mitigation equals 'No known mitigation'.
-    #     """
-    #     return [
-    #         "title",
-    #         "description",
-    #         "url",
-    #         "severity",
-    #         "impact",
-    #         "mitigation",
-    #     ]
-    # Dedupe for v2 based on default dedupe values
-    # def get_dedupe_fields(self) -> list[str]:
-    #     """
-    #     Return the list of fields used for deduplication in the Aqua Parser V2.
-    #
-    #     Fields:
-    #     - title: Created by combining the finding's cve and file_path
-    #     - description: Text describing finding
-    #     """
-    #     #NOTE: vulnerability_ids is not provided by parser
-    #     return [
-    #         "title",
-    #         "description",
-    #     ]
```

Comment:
While this currently isn't utilized by DefectDojo, we have an intern making these changes so that, once they are all done, they can be used. This portion should not be removed though, I agree, it's not doing much currently.